test/controllers/IndexControllerSpec.scala

```diff
@@ -18,278 +18,162 @@ package controllers
 
+      when(mockUserAnswerService.set(any())(any(), any())).thenReturn(Future(true))
       val result = controller.onPageLoad()(fakeRequest)
 
       status(result) mustEqual OK
+      titleOf(contentAsString(result)) mustBe
+        "Add information about the charity - Register your charity’s details with HMRC - GOV.UK"
+
       verify(mockUserAnswerService, times(1)).get(any())(any(), any())
+
     }
 
     "redirect to the session expired page if no valid session id for switchover journey" in {
 
-      val requestWithoutSession = FakeRequest()
-      when(mockUserAnswerService.get(any())(any(), any())).thenReturn(
-        Future.successful(
-          Some(emptyUserAnswers.set(Section1Page, true).flatMap(_.set(Section2Page, false)).success.value)
-        )
-      )
-      when(mockUserAnswerService.set(any())(any(), any())).thenReturn(Future.successful(true))
+      val userAnswers =
+        emptyUserAnswers
+          .set(Section1Page, true)
+          .flatMap(_.set(Section2Page, false))
+          .success
+          .value
 
-      val result = controller.onPageLoad()(requestWithoutSession)
+      when(mockUserAnswerService.get(any())(any(), any())).thenReturn(Future(Some(userAnswers)))
+      when(mockUserAnswerService.set(any())(any(), any())).thenReturn(Future(true))
+
+      val result = controller.onPageLoad()(fakeRequestNoSessionId)
 
       status(result) mustEqual SEE_OTHER
       verify(mockUserAnswerService, times(1)).get(any())(any(), any())
       verify(mockUserAnswerService, never).set(any())(any(), any())
     }
 
-    "redirect to the task list without calling save4later service if external test is enabled" in {
-
-      val app =
-        new GuiceApplicationBuilder()
-          .configure("features.isExternalTest" -> "true")
-          .overrides(
-            bind[CharitiesSave4LaterService].toInstance(service),
-            bind[UserAnswerService].toInstance(mockUserAnswerService),
-            bind[CharitiesShortLivedCache].toInstance(mockCharitiesShortLivedCache),
-            bind[AuthIdentifierAction].to[FakeAuthIdentifierAction]
-          )
-          .build()
-
-      when(mockUserAnswerService.get(any())(any(), any())).thenReturn(Future.successful(None))
-      when(mockCharitiesShortLivedCache.fetchAndGetEntry[Boolean](any(), any())(any(), any(), any()))
-        .thenReturn(Future.successful(Some(true)))
-      when(mockUserAnswerService.set(any())(any(), any())).thenReturn(Future.successful(true))
-
-      val controller: IndexController = app.injector.instanceOf[IndexController]
-
-      val result = controller.onPageLoad()(fakeRequest)
-
-      status(result) mustEqual OK
-      verify(mockUserAnswerService, times(1)).get(any())(any(), any())
-      verify(mockUserAnswerService, times(1)).set(any())(any(), any())
-      verify(mockCharitiesShortLivedCache, never).fetchAndGetEntry[Boolean](any(), any())(any(), any(), any())
-    }
-
-    "For keepalive" in {
+    "for keepalive" in {
```

Comment:
This should also follow the when -> must -> in. Currently it is when -> in which doesn't read well.